packages/worksheet/src/pages/CreateWorksheet.js

```diff
@@ -26,8 +29,17 @@ export default function CreateWorksheet({ footerLinks, appName }) {
   const [limit, setLimit] = React.useState({});
   const [alertMessage, setAlertMessage] = React.useState();
   const [createType, setCreateType] = React.useState("create");
+  const [worksheetStartTime, setWorksheetStartTime] = React.useState();
   const navigate = useNavigate();
 
+  React.useEffect(() => {
+    const telemetryData = telemetryFactory.interact({
+      appName,
+      type: "Worksheet-Create",
+    });
+    capture("INTERACT", telemetryData);
+  }, []);
```

Comment:
pls check if any scope can be added for the effect.

Reply:
If you want to run an effect and clean it up only once (on mount and unmount), you can pass an empty array ([]) as a second argument. This tells React that your effect doesn’t depend on any values from props or state, so it never needs to re-run. This isn’t handled as a special case — it follows directly from how the dependencies array always works.